scripts/export-import-test.py

```diff
@@ -0,0 +1,121 @@
+import sys
+import argparse
+import subprocess
+import os
+import shutil
+
+
+def getVersion(executablePath):
+    try:
+        result = subprocess.run(
+            [executablePath, "--version"],
+            capture_output=True,
+            text=True,
+            check=True
+        )
+        output = result.stdout.strip()
+        if output.startswith("Kuzu "):
+            return output.split(" ", 1)[1]
+        else:
+            print("Unexpected version format:", output)
+            return None
+    except subprocess.CalledProcessError as e:
+        print(f"Error running '{executablePath} --version': {e}")
+        return None
+
+
+def runCommand(cmd, cwd=None, env=None, capture_output=False):
+    if isinstance(cmd, str):
+        cmd = cmd.split()
+
+    print(f"> Running: {' '.join(cmd)} (cwd={cwd})")
+
+    if capture_output:
+        result = subprocess.run(cmd, cwd=cwd, env=env, text=True, capture_output=True, check=True)
+        print(result.stdout)
+        return result.stdout.strip()
+    else:
+        process = subprocess.Popen(
+            cmd,
+            cwd=cwd,
+            env=env,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            bufsize=1,
+            universal_newlines=True,
+            stdin=subprocess.DEVNULL
+        )
+
+        for line in process.stdout:
+            print(line, end='')
+
+        process.stdout.close()
+        retcode = process.wait()
+        if retcode != 0:
+            raise subprocess.CalledProcessError(retcode, cmd)
+        return None
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Export DBs from datasetDir using commitA and test in commitB"
+    )
+    parser.add_argument("commitA", help="Git commit to export databases from")
```

Comment:
Can you add descriptive flags: `--base-commit` `--test-commit` `--dataset-dir` `--output-dir`